### ISSUE TITLE
Ambiguity error on pagerank

### ISSUE THREAD

**Issue by Tokazama** [sbromberger/SimpleWeightedGraphs.jl#60]
Had the following ambiguity error just pop up.

```julia
julia> SimpleWeightedGraphs.pagerank(g)
ERROR: MethodError: pagerank(::SimpleWeightedDiGraph{Int64,Float64}, ::Float64, ::Int64, ::Float64) is ambiguous. Candidates:
  pagerank(g::SimpleWeightedDiGraph, α, n, ϵ) in SimpleWeightedGraphs at /Users/zchristensen/.julia/packages/SimpleWeightedGraphs/yU
Frc/src/overrides.jl:22
  pagerank(g::AbstractGraph{U}, α, n::Integer, ϵ) where U<:Integer in LightGraphs at /Users/zchristensen/.julia/packages/LightGraphs
/HsNig/src/centrality/pagerank.jl:21
Possible fix, define
  pagerank(::SimpleWeightedDiGraph{U<:Integer,U} where U<:Real, ::Any, ::Integer, ::Any)
S
```

**Comment by sbromberger** [comment]
What's `g`?

**Comment by sbromberger** [comment]
I can reproduce with
```
g = SimpleWeightedDiGraph(path_graph(10), 3)
```

**Comment by sbromberger** [comment]
This was fixed by #55. We need to tag a new release but the master branch passes all tests.
